### PR TITLE
add example for solid model

### PR DESCRIPTION
**Pull request by roosre**
Add an example how to generate a solid model and shape it.

Additional change: add unit tests for Edge Properties Lamina, FabricWithAngel, SubShape and TaperEdge

### CODE REVIEW

File: examples/modeling_features/020-solid_model.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    ExampleKeys.CUT_OFF_GEOMETRY_SOLID_MODEL
+)
+solid_model.create_cut_off_geometry(
+    name="Cut-off Geometry",
+    cad_geometry=cutoff_virtual_geom,
+    orientation_type=CutOffGeometryOrientationType.UP,
+)
+
+model.update()
+plot_model_with_geometry(cutoff_cad_geom)
```
Comment (greschd) [inline examples/modeling_features/020-solid_model.py:185]: Nice example!

I'm wondering if we should show how to plot on a partial mesh in the same example, e.g. plotting the thickness of a single analysis ply.

**Review by greschd — APPROVED** [review]

File: examples/modeling_features/020-solid_model.py (hunk 1 above)
Comment (roosre) [reply examples/modeling_features/020-solid_model.py:185]: Done.
